gnovm/pkg/gnolang/preprocess.go

```diff
@@ -2472,6 +2472,9 @@ func convertIfConst(store Store, last BlockNode, x Expr) {
 
 func convertConst(store Store, last BlockNode, cx *ConstExpr, t Type) {
 	if t != nil && t.Kind() == InterfaceKind {
+		if cx.T != nil && t != nil {
```

Comment:
```suggestion
		if cx.T != nil {
```
the line above checks that t is not nil.